vendor/github.com/containers/common/libimage/import.go

```diff
@@ -61,7 +63,8 @@ func (r *Runtime) Import(ctx context.Context, path string, options *ImportOption
 	u, err := url.ParseRequestURI(path)
 	if err == nil && u.Scheme != "" {
 		// If source is a URL, download the file.
-		file, err := r.downloadFromURL(path)
+		fmt.Printf("Downloading from %q\n", path)
```

Comment:
Is this a bug?

Reply:
@vrothberg PTAL

Reply:
It's preserving previous behavior, see https://github.com/containers/common/commit/2f3c4bcdfdcf.  But I am for changing the behavior. Libraries should not print on stdout.

Reply:
Well that should happen in a separate PR.

Reply:
Agreed, that's what I also wrote in the PR of https://github.com/containers/common/commit/2f3c4bcdfdcf.  It may require changes to Podman to preserve behavior but I did not investigate.